### PR TITLE
Allow repeatable annotations

### PR DESCRIPTION
**Pull request by Sanne**
- https://hibernate.atlassian.net/browse/HSEARCH-2574

### CODE REVIEW

**Comment by Sanne** [comment]
I pushed an improvement.

File: documentation/src/main/asciidoc/mapping.asciidoc [hunk 1]
```diff
@@ -440,20 +441,21 @@ document id, it is recommended to declare another field specifically for sorting
...
 ----
 @Entity
 @Indexed(index = "Book")
 public class Book {
-    @Fields( {
-            @Field,
-            @Field(name = "summary_forSort", analyze = Analyze.NO, store = Store.YES)
-    } )
+
+    @Field,
```
Comment (yrodiere) [inline documentation/src/main/asciidoc/mapping.asciidoc]: There's an extra comma here.

Comment (Sanne) [inline documentation/src/main/asciidoc/mapping.asciidoc]: thanks, fixed!

**Comment by Sanne** [comment]
> Here's an error I noticed while reading through the changes.

From now on, you shall be remembered as "eagle eye Yoann" :+1:

**Comment by Sanne** [comment]
I noticed as well, but came to the same conclusions. `TikaBridge` has also an uncertain future IMO: I like the functionality it provides but it brings too many dependencies; I'd like to use it as an example of how we might be able to tightly integrate with "advanced bridges", including possibly having their own annotations, yet have these live in a separate module so that we can isolate the dependencies properly. This might be a strong requirement for producing Jigsaw modules..

In short, I'd not see the benefit on opening new tickets for now.

**Comment by yrodiere** [comment]
Merged! Thanks.
And ok to look at TikaBridge again for HSearch 6.0.
